contrib/kubespray/roles/requirement/infra/tasks/ubuntu.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: "Infra-Task-1-[1/2] Check whether the vm can access to docker.io"
+  raw: nslookup index.docker.io
+  register: infra_nslookup_docker_io
+  failed_when: false
+  changed_when: false
+  check_mode: false
+  environment: {}
+
+- name: "Infra-Task-1-[2/2] Check whether the vm can access to docker.io"
```

Comment:
the numbering is confusing, just using 1.1, 1.2, 2.1 is enough
the total number is useless and you need to modify all occurrences if you add a new task later

Reply:
Done. With title infra-1.1, infra-1.2, infra-2.1